plugins/identity-shiro/src/main/java/org/opensearch/identity/shiro/ShiroSubject.java

```diff
@@ -10,28 +10,28 @@
 
 import java.security.Principal;
 import java.util.Objects;
-
+import java.util.Optional;
 import org.opensearch.identity.Subject;
 import org.opensearch.identity.tokens.AuthToken;
 
 /**
- * Subject backed by Shiro
+ * A Subject backed by the Identity Shiro Plugin and wrapping a shiro.subject implementing Subject
  *
  * @opensearch.experimental
  */
 public class ShiroSubject implements Subject {
     private final ShiroTokenManager authTokenHandler;
-    private final org.apache.shiro.subject.Subject shiroSubject;
+    private final org.apache.shiro.subject.Subject wrappedSubject;
```

Comment:
Super nit, but may be `delegate` is a better name?
```suggestion
    private final org.apache.shiro.subject.Subject delegate;
```